panel/_styles/widgets.css

```diff
@@ -139,3 +139,203 @@ progress:not([value])::before {
   from {background-position: 0%}
   to {background-position: 100%}
 }
+
+div.loader:after {
+  content: "";
+  border-radius: 50%;
+  -webkit-mask-image: radial-gradient(transparent 50%, rgba(0, 0, 0, 1) 54%);
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  position: absolute;
+}
+
+div.loader.dark:after {
+  background: conic-gradient(
+    #0f0f0f 0 100%
+  ) !important;
```

Comment:
What's the reason for using `!important` here and in the other cases?

Reply:
I'll have to look again, I **think** I can remove those now.